tests/generic_config_updater/gu_utils.py

```diff
@@ -326,13 +326,25 @@ def get_asic_name(duthost):
     return asic
 
 
-def is_valid_platform_and_version(duthost, table, scenario):
+def is_valid_platform_and_version(duthost, table, scenario, field=None, operation=None):
     asic = get_asic_name(duthost)
-    os_version = duthost.os_version
     if asic == "unknown":
         return False
+    gcu_conf = get_gcu_field_operations_conf()
+    if field:
+        if field not in (
+            gcu_conf["tables"][table]["validator_data"]
```

Comment:
>gcu_conf["tables"][table]["validator_data"] [](http://example.com/codeflow?start=12&length=43)

It's confusing to write multi-dimension index in 2 lines. Suggest to use a temporary variable, and reuse it.